### PR TITLE
release prep

### PR DESCRIPTION
**Pull request by amaredeus**
preparing for minor release

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -40,7 +40,6 @@ build/Release
 # Dependency directories
 node_modules/
 jspm_packages/
-package-lock.json
```
Comment (nonsensecodes) [inline .gitignore:43]: Hey, @amaredeus, don't we want this to be ignored, in the case that someone uses npm to install dependencies and run scripts?

**Review by nonsensecodes — CHANGES_REQUESTED** [review]
Just one inquiry, @amaredeus.

File: .gitignore (hunk 1 above)
Comment (amaredeus) [reply .gitignore:43]: This came up during a troubleshooting session with @geositta. The concern was that there should only be 1 lock file for the build out, either yarn or npm, if both are included that might cause a security vulnerability. In the case of birdman, the npm lock file was present and completely iginored by git. Even though he reverted all changes using git, the npm lock file was still present. Removing the file from .gitignore will make sure git sees the file and we can reject it if it gets included in a future PR

Comment (nonsensecodes) [reply .gitignore:43]: Gotcha. Okay. Thank you for clarifying this.

**Review by nonsensecodes — APPROVED** [review]
Good to go!

**Pull request merged**